server/backend/testspg/crud/validator/test_forest_client_validator.py

```diff
@@ -76,57 +87,23 @@ def test_forest_client_number_exists(
     client_id_to_test,
     expcted_result,
     mock_forest_client,
+    forest_client_integration_service,
 ):
-    request_dict = {**ACCESS_GRANT_FOM_DEV_AR_00000001_IDIR, "forest_client_number": client_id_to_test}
     mock_forest_client.find_by_client_number.return_value = (
         __to_mock_forest_client_return(
-            request_dict["forest_client_number"],
+            client_id_to_test,
             expcted_result["api_client_status_code"],
         )
     )
 
-    request = FamUserRoleAssignmentCreate(**request_dict)
-    validator = UserRoleValidator(request)
-    result = validator.forest_client_number_exists()
-
-    mock_forest_client.find_by_client_number.assert_called_with(client_id_to_test)
-    assert result is expcted_result["exists"]
-
-
-@pytest.mark.parametrize(
-    "client_id_to_test, expcted_result",
-    [
-        (FC_NUMBER_LEN_TOO_SHORT, {"api_client_status_code": None}),
-        (FC_NUMBER_NOT_EXISTS, {"api_client_status_code": None}),
-        (FC_NUMBER_EXISTS_ACTIVE_00000001, {"api_client_status_code": "ACT"}),
-        (FC_NUMBER_EXISTS_DEACTIVATED, {"api_client_status_code": "DAC"}),
-        (FC_NUMBER_EXISTS_DECEASED, {"api_client_status_code": "DEC"}),
-        (FC_NUMBER_EXISTS_RECEIVERSHIP, {"api_client_status_code": "REC"}),
-        (FC_NUMBER_EXISTS_SUSPENDED, {"api_client_status_code": "SPN"}),
-    ],
-)
-def test_get_forst_client(
-    client_id_to_test,
-    expcted_result,
-    mock_forest_client,
-):
-    request_dict = {**ACCESS_GRANT_FOM_DEV_AR_00000001_IDIR, "forest_client_number": client_id_to_test}
-    mock_forest_client.find_by_client_number.return_value = (
-        __to_mock_forest_client_return(
-            request_dict["forest_client_number"],
-            expcted_result["api_client_status_code"],
-        )
+    # find forest client number
+    forest_client_validator_return = (
```

Comment:
Since this is not a return from validator, maybe the variable can be "forest_client_search_return"?